tests/elasticache.rs

```diff
@@ -2,7 +2,7 @@
 
 extern crate rusoto;
 
-use rusoto::elasticache::{ElastiCacheClient, DescribeCacheClustersMessage};
+use rusoto::elasticache::{Elasticache, ElastiCacheClient, DescribeCacheClustersMessage};
```

Comment:
Interesting...that definitely shouldn't have gotten past CI. Good catch though!

Oh, wait, no. Only lib tests are done in CI. It would be handy to at least build integration tests to verify them though. There's a new flag for Cargo coming that allows passing `--tests` to `cargo build` to build tests, but I don't think it will be dropping until 1.19. Hrm... I guess the only good way to catch that is locally.